### DISCUSSION TITLE
Please Approval of WalletConnect registry 🙏,  We  have update Universal link

### DISCUSSION THREAD

**Discussion by SlopeFinance** [WalletConnect/walletconnect-monorepo#1078] [Developer Support] [answered]
Hi there!
We  have update `Universal link ` and submitted an application to register our wallet "Slope Wallet"
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance.

**Comment by chadyj** [comment, marked as answer]
Done!

**Reply by chenfan1227** [reply]
你好呀！
我们已更新Universal link 并提交了注册我们的钱包“hideout wallet”
的申请，请您批准该申请，以便我们的用户可以尽快访问 WallectConnect。
提前致谢。

**Reply by chenfan1227** [reply]
Hi there!
We have update Universal link and submitted an application to register our wallet “hideout wallet”
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance.
